tests/suites/test_suite_bignum_core.function

```diff
@@ -5,6 +5,154 @@
 #include "constant_time_internal.h"
 #include "test/constant_flow.h"
 
+/** Verifies mbedtls_mpi_core_add().
+ *
+ * \param[in] A       Little-endian presentation of the left operand.
+ * \param[in] B       Little-endian presentation of the right operand.
+ * \param limbs       Number of limbs in each MPI (\p A, \p B, \p S and \p X).
+ * \param[in] S       Little-endian presentation of the expected sum.
+ * \param carry       Expected carry from the addition.
+ * \param[in,out] X   Temporary storage to be used for results.
+ *
+ * \return  1 if mbedtls_mpi_core_add() passes this test, otherwise 0.
```

Comment:
My bad. We do have a habit of returning pass/fail booleans rather than status codes from auxiliary functions. I'm afraid I started this trend in PSA tests, where a status code was `psa_status_t` and thus it was natural to give the type `int` its habitual C meaning of true/false indicating ok/not-ok. Then we've carried this over to non-PSA tests where `int` is ambiguous. And now we have a mix of conventions.

Reply:
In the library this would be a blocker - mixed use of "0 means success" and "truthy (non-zero) means success" have been famously confusing in OpenSSL - but in test code we can live with it.